docs: update uninstallation guide (#2281)
codenvy/che-docs: modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc

:_content-type: PROCEDURE
:navtitle: Uninstalling {prod-short} by using {prod-cli}
:keywords: administration guide, uninstalling-che, uninstalling-che-after-chectl-installation
:page-aliases: installation-guide:uninstalling-che-after-chectl-installation

[id="uninstalling-{prod-id-short}-by-using-{prod-cli}_{context}"]
= Uninstalling {prod-short} by using `{prod-cli}`

To uninstall an instance of {prod} that was installed using the `{prod-cli}` tool on {platforms-name}:

.Prerequisites

* The `{prod-cli}` tool was used to install the {prod-short} instance on {platforms-name}.
* The `{prod-cli}` and `{orch-cli}` tools are available.

.Procedure

. Obtain the name of the {prod-short} namespace:

+
`$ {orch-cli} get checluster --all-namespaces -o=jsonpath="{.items[*].metadata.namespace}"`

. Remove the {prod-short} instance from the {orch-namespace}:

+
`$ {prod-cli} server:delete -n __<{orch-namespace}>__`

+

> include::example$snip_{project-context}-note-uninstalling-in-minicube-cluster.adoc[]

Reviewer: ```suggestion
include::example$snip_{project-context}-note-uninstalling-in-minikube-cluster.adoc[]
```

:_content-type: PROCEDURE
:navtitle: Uninstalling {prod-short} by using {prod-cli}
:keywords: administration guide, uninstalling-che, uninstalling-che-after-chectl-installation
:page-aliases: installation-guide:uninstalling-che-after-chectl-installation

[id="uninstalling-{prod-id-short}-by-using-{prod-cli}_{context}"]
= Uninstalling {prod-short} by using `{prod-cli}`

To uninstall an instance of {prod} that was installed using the `{prod-cli}` tool on {platforms-name}:

.Prerequisites

* The `{prod-cli}` tool was used to install the {prod-short} instance on {platforms-name}.
* The `{prod-cli}` and `{orch-cli}` tools are available.

.Procedure

. Obtain the name of the {prod-short} namespace:

+
`$ {orch-cli} get checluster --all-namespaces -o=jsonpath="{.items[*].metadata.namespace}"`

. Remove the {prod-short} instance from the {orch-namespace}:

+
`$ {prod-cli} server:delete -n __<{orch-namespace}>__`

+
[NOTE]
====

> If uninstalling {prod-short} from a Minikube cluster, optionally remove the `dex` namespace.

Reviewer: ```suggestion
If uninstalling {prod-short} from a Minikube cluster: remove the `dex` and Che namespaces.
```

> `$ {orch-cli} delete namespaces dex`

Reviewer: ```suggestion
`$ {orch-cli} delete namespaces dex __<{orch-namespace}>__`
```
